.github/workflows/integration-tests-design-system.yml

```diff
@@ -0,0 +1,52 @@
+name: Run integration tests on design system
+
+on: workflow_call
+
+jobs:
+  run-integration-tests-design-system:
+    name: Run integration tests on design system
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Would it be possible to 'DRY this up' by using a matrix strategy, as you've already done with MiniTest?

That way we'd only need one `.github/workflows/integration-tests.yml` file, right?

Reply:
Definitely. I went for this version because I thought it could be easier to spot when tidying up after design system is all ported over, but maybe it doesn't make much difference.